civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc

```diff
@@ -15,14 +15,14 @@ function civihr_employee_portal_features_views_default_views() {
   $view->description = '';
```

Comment:
Please fix the permission for this file too

Reply:
The ticket description has been updated with details about the permissions

Reply:
Was this fixed?

Reply:
Yes. The first time the PR was sent it was changed to 755, so I asked Rene to revert it to 644. With the last updates, this  was not necessary anymore